src/components/common/copy-to-clipboard-button/copy-to-clipboard-button.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment, useCallback, useRef, useState } from 'react'
+import { Button, Overlay, Tooltip } from 'react-bootstrap'
+import { Variant } from 'react-bootstrap/types'
+import { Trans, useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon } from '../fork-awesome/fork-awesome-icon'
+
+export interface CopyToClipboardButtonProps {
+  content: string
+  size?: 'sm' | 'lg'
+  variant?: Variant
+}
+
+export const CopyToClipboardButton: React.FC<CopyToClipboardButtonProps> = ({ content, size = 'sm', variant = 'dark' }) => {
+  const { t } = useTranslation()
+  const button = useRef<HTMLButtonElement>(null)
+  const [showCopiedTooltip, setShowCopiedTooltip] = useState(false)
+
+  const copyToClipboard = useCallback((content: string) => {
+    navigator.clipboard.writeText(content).then(() => {
+      setShowCopiedTooltip(true)
+      setTimeout(() => { setShowCopiedTooltip(false) }, 2000)
+    }).catch(() => {
+      console.error("couldn't copy")
```

Comment:
Maybe show an error tooltip instead of silently failing?

Reply:
done

Reply:
maybe CopyableField should do the same?

Reply:
done